lint-staged.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  '*.+(js|jsx|ts|tsx)': ['yarn lint'],
+  // '*.+(js|jsx|ts|tsx)': ['eslint', 'yarn test:precommit'],
```

Comment:
shouldn't we run `eslint` only on the changed files to make it faster?

can you remove the commented out line?

Reply:
but `yarn lint` will already run eslint, won't uncommenting just add *more* steps?

given that it's merged, IMHO feel free to test locally and make a new PR with whatever config will be the fastest while doing all needed checks 😂

Reply:
it wasn't merged when I made the comment 😅
I will check it locally later today and make another PR if it is faster

Reply:
sorry there wasn't electricity 😅😅
also @merodiro I have no idea how to make eslint run on the changed files only. the commented out line wouldn't do any harm (as far as my experience go)